scripts/Thaumcraft-04-Artifice.zs

```diff
@@ -277,9 +277,9 @@ mods.thaumcraft.Research.addArcanePage("BASICARTIFACE", <Thaumcraft:ItemBaubleBl
 
 // --- Mirrored Glass
 mods.thaumcraft.Arcane.addShaped("BASICARTIFACE", <Thaumcraft:ItemResource:10>, "aer 30, terra 30, ignis 30,", [
-[<ore:bucketEnder>, <ore:gemMercury>, <ore:bucketEnder>],
+[<ore:gemMercury>, <gregtech:gt.blockgem1:8>, <ore:gemMercury>],
```

Comment:
If I add the cells, it doesn't give back the empty cells, and I didn't want to bother glease about extending the ward focus thing. The block is an ender pearl block.

Reply:
Why not try ender cell once? To my knowledge it will properly give back the empty cells. Either normal IC2 cells or universal fluid container should work.

Reply:
As I said, it doesn't. Also, there's some weird thing going on with the comments being split on this, maybe you didn't see that?

Reply:
ok change it to somethings like plate. A block not fit. Or dense plate or double or so

Reply:
Weird it doesn't. It can be easily extended to support returning empty cells. It is a lot less complex than wand focus, because we need to preserve the NBT data on focus, where cells have none.

Reply:
I set it to the cells, so I hope you can help with that when you have some time Glease. BTW, it's the universal cells, there's no normal cell version.